src/main.rs

```diff
@@ -68,6 +72,17 @@ include:
     http://llvm.org/docs/LibFuzzer.html#dictionaries\
 ";
 
+const BUILD_BEFORE_HELP: &'static str = "\
+By default fuzz targets are built with optimizations equivalent to
+`cargo build --release`, but with debug assertions and overflow checks enabled.
```

Comment:
> but with debug assertions and overflow checks enabled

which code in this pr is responsible for enabling debug assertions with release mode by default?